### ISSUE TITLE
Training Yolo doesnt improve precision and Recall (Stuck at 0)

### ISSUE THREAD

**Issue by take2rohit** [Tessellate-Imaging/Monk_Object_Detection#18]
I have exactly used codes as mentioned [here](https://github.com/Tessellate-Imaging/Monk_Object_Detection/tree/master/example_notebooks/7_yolov3) in examples of yolo. I have tested using football dataset and grapes dataset as used in provided notebooks. I am not getting the inference results.

The code on colab implemented by me can me found [HERE](https://colab.research.google.com/drive/11-yoIrPis92jRZLfNvwJSwLRgwMkyA_V)
Please look into it.

![image](https://user-images.githubusercontent.com/40535193/77532695-64f5ee00-6ebb-11ea-8a0b-3ef7670b060c.png)

**Comment by abhi-kumar** [comment]
Try reducing the learning rate and retrain.

**Comment by abhi-kumar** [comment]
@take2rohit any updates on this?

**Comment by take2rohit** [comment]
I will let you know by today

**Comment by abhi-kumar** [comment]
Sure.

**Comment by take2rohit** [comment]
Trained even with reduced learning rate `lr=1e-5` still no change
Training isn't learning anything

**Comment by mihir135** [comment]
Same issue here. mAP, precision, and recall stuck at 0 for all the epochs. I tried to train with 50 epochs on custom dataset.

**Comment by abhi-kumar** [comment]
The issue was with certain network architecture conversions, YoloV3 requires certain changes in config files and monk automated it, but this automation had certain errors which have now been rectified completely

**Comment by abhi-kumar** [comment]
Closing due to inactivity
